### DISCUSSION TITLE
How to show an image url on view resource

### DISCUSSION THREAD

**Discussion by Katznicho** [filamentphp/filament#10661] [Help]
Am working on some project but the images are actually stored in firebase show we only resource url .How can I show them in my filament project I also want to get access to the current product on the vie w page
![image](https://github.com/filamentphp/filament/assets/57605587/a8c89703-1ae8-4de6-b941-b41c24d89943)

**Comment by leandrocfe** [comment]
You can use a view field to render the images using a blade foreach
https://filamentphp.com/docs/3.x/forms/fields/custom#view-fields

You can use a infolist + image entry

https://filamentphp.com/docs/3.x/panels/resources/viewing-records#using-an-infolist-instead-of-a-disabled-form
https://filamentphp.com/docs/3.x/infolists/entries/image#overview
